src/Task/CloneSite.php

```diff
@@ -134,9 +134,9 @@ protected function getDestinationSiteStatus() {
       $drushCmd->dir(DocrootResolver::getDocroot($this->getDestination()));
 
       $drushCmd
-        ->arg('status')
+        ->arg('core-status')
```

Comment:
Sometimes you are replacing 'status' by 'core-status'. Sometimes not. Is that right?

Reply:
No, I just forgot to do that in the other places. Will use core-status in the other places as well, I prefer full commands to short-hand in code.